### PR TITLE
cross/build-rootfs.sh: fix default UbuntuPackages for default BuildArch

### LINKED ISSUE #7643: ARM cross-compile fails when no args given to cross/build-rootfs.sh
Steps to reproduce:
1. `cross/build-rootfs.sh`
2. `./build.sh arm debug cross`

Expected result:
- CoreCLR for ARM builds successfully

Observed result:
- build fails because `lldb-3.6-dev` was not installed when creating chroot/rootfs

This is due to `cross/build-rootfs.sh` requiring `arm` being explicitly called out as the `BuildArch` arg in order for the `__LLDB_Package` var to be added to `__UbuntuPackages`.

#7644 fixes this.

### PR DESCRIPTION
**Pull request by stevedesmond-ca**
Installs lldb-3.6-dev if no BuildArch arg is specified, since the default BuildArch (ARM) requires lldb-3.6-dev.

Fixes #7643.

### CODE REVIEW

**Comment by jkotas** [comment]
cc @janvorli

File: cross/build-rootfs.sh [hunk 1]
```diff
@@ -17,10 +17,10 @@ __BuildArch=arm
 __UbuntuArch=armhf
 __UbuntuRepo="http://ports.ubuntu.com/"
 __UbuntuPackagesBase="build-essential libunwind8-dev gettext symlinks liblttng-ust-dev libicu-dev"
-__UbuntuPackages="$__UbuntuPackagesBase"
 if [ -z "$LLVM_ARM_HOME" ]; then
     __LLDB_Package="lldb-3.6-dev"
 fi
+__UbuntuPackages="$__UbuntuPackagesBase $__LLDB_Package"
```
Comment (janvorli) [inline cross/build-rootfs.sh]: Can you please delete setting the `__UbuntuPackages` in the case for arm and arm-softfp below? It is not needed there anymore.

Comment (stevedesmond-ca) [inline cross/build-rootfs.sh]: Done -- there's actually a larger refactor (including tests, see #7645) that I see as useful: splitting the argument parsing from the logic to set the environment variables correctly.

Would that be better as part of this PR, or a new/separate PR once the specific issue for this PR is fixed?

**Comment by janvorli** [comment]
@stevedesmond-ca Thank you for fixing it!

**Pull request merged**